src/test/java/org/broadinstitute/hellbender/utils/help/TestWDLTool.java

```diff
@@ -0,0 +1,187 @@
+package org.broadinstitute.hellbender.utils.help;
+
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.barclay.argparser.PositionalArguments;
+import org.broadinstitute.barclay.argparser.WorkflowProperties;
+import org.broadinstitute.barclay.argparser.WorkflowInput;
+import org.broadinstitute.barclay.argparser.WorkflowOutput;
+import org.broadinstitute.barclay.help.DocumentedFeature;
+import org.broadinstitute.hellbender.cmdline.TestProgramGroup;
+
+import java.io.File;
+import java.util.List;
+
+/**
+ * NOTE: this file needs to live in a separate package from the doc tests, otherwise all of the docs tests
+ * will pick it up as a command line program, which will change the outputs.
+ *
+ * CommandLineProgram test tool for testing WDL generation. Contains various combinations of
+ * commandline argument and workflow input/outputs with companion resources:
+ *
+ *  scalar/array
+ *  file/non-file
+ *  required/optional
+ */
+@CommandLineProgramProperties(
+        summary = TestWDLTool.SUMMARY,
+        oneLineSummary = TestWDLTool.ONE_LINE_SUMMARY,
+        programGroup = TestProgramGroup.class)
+@WorkflowProperties(memory ="8G")
+@DocumentedFeature(groupName = TestWDLTool.GROUP_NAME)
+public class TestWDLTool {
+
+    public static final String SUMMARY = "WDL Test Tool";
+    public static final String ONE_LINE_SUMMARY = "WDL Test Tool to test WDL Generation";
+    public static final String GROUP_NAME = "WDL feature group name";
+
+    @PositionalArguments(doc = "Positional args doc")
+    @WorkflowInput(requiredCompanions={"posDictionary", "posIndex"})
+    public List<File> positionalListFileInput;
+
+    // required Files
+    @Argument(fullName = "requiredScalarFileInput",
+            shortName = "requiredScalarFileInput",
+            doc = "requiredScalarFileInput doc",
+            optional = false)
+    @WorkflowInput(requiredCompanions={"requiredScalarFileInputDictionary", "requiredScalarFileInputIndex"}, localizationOptional = true)
+    public File requiredScalarFileInput;
+
+    @Argument(fullName = "requiredListFileInput",
+            shortName = "requiredListFileInput",
+            doc = "requiredListFileInput doc",
+            optional = false)
+    @WorkflowInput(requiredCompanions={"requiredListFileInputDictionary", "requiredListFileInputIndex"}, localizationOptional = true)
```

Comment:
Can you include at least one `@WorkflowInput` / `@WorkflowOutput` in this test class that does not have any companion files?

Reply:
Done.